schemas/com.sendgrid/bounce/jsonschema/2-0-0

```diff
@@ -0,0 +1,107 @@
+      "type": "string",
+      "minLength": 22,
+      "maxLength": 48
+    },
+    "smtp-id": {
+      "description": "a unique ID attached to the message by the originating system",
+      "type": "string"
+    },
+    "reason": {
+      "description": "any sort of error response returned by the receiving server that describes the reason this event type was triggered",
+      "type": "string"
+    },
+    "tls": {
+      "description": "indicates whether TLS encription was used in sending this message. For more information about TLS, see the SendGrid TLS Glossary page.",
+      "type": ["string", "integer"]
+    },
+    "status": {
+      "description": "status code string. Corresponds to HTTP status code - for example, a JSON response of 5.0.0 is the same as a 500 error response.",
+      "type": "string"
+    },
+    "cert_error": {
+      "description": "no longer found in the SendGrid documentation; possibly deprecated",
+      "type": "string"
+    },
+    "category": {
+      "description": "Categories are custom tags that you set for the purpose of organizing your emails. Categories can be set as an array or string, and they will be returned as such when posted in your event endpoint.",
+      "type": ["array", "string"],
+      "items": {
+        "type": "string"
+      }
+    },
+    "asm_group_id": {
+      "description": "The ID of the unsubscribe group the recipient’s email address is included in. ASM IDs correspond to the ID that is returned when you create an unsubscribe group.",
+      "type": "integer",
+      "maximum": 32767,
+      "minimum": 0
+    },
+    "type": {
+      "description": "type of bounce, eg Bounce/Blocked/Expired",
+      "type": "string"
+    },
+    "sg_message_id": {
+      "description": "a unique, internal SendGrid ID for the message. The first half of this is pulled from the smtp-id.",
+      "type": "string"
+    },
+    "marketing_campaign_id": {
+      "description": "For emails sent through our Marketing Campaigns feature, we add Marketing Campaigns specific parameters to the Event Webhook. Both marketing_campaign_name and marketing_campaign_id are displayed as unique arguments in the event data.",
+      "type": "integer"
+    },
+    "marketing_campaign_name": {
+      "description": "For emails sent through our Marketing Campaigns feature, we add Marketing Campaigns specific parameters to the Event Webhook. Both marketing_campaign_name and marketing_campaign_id are displayed as unique arguments in the event data.",
+      "type": "string"
+    },
+    "marketing_campaign_version": {
+      "description": "displayed in the event data for emails sent as part of an A/B Test. The value for marketing_campaign_version are returned as A, B, C, etc.",
+      "type": "string"
+    },
+    "marketing_campaign_split_id": {
+      "description": "marketing campaign split id",
+      "type": "integer"
+    }
+  },
+  "additionalProperties": true
```

Comment:
Generally, `additionalProperties: false` is more preferable option as it makes a schema more explicit. It also totally makes sense to make `additionalProperties: true` when you don't have a control over data produced by 3rd party providers. However, in webhooks we're schemaing the _output_ format, not _input_, so if we can assume that webhook can strip away all unkown fields - we certainly can make it `additionalProperties: false`. This is not a request, just food for thought.